components/BookDetails/index.js

```diff
@@ -24,14 +24,12 @@ export default function BookDetails({ book }) {
       <StyledDetailsPage>
         <StyledFrame>
           <div>
-            <StyledImage>
-              <Image
-                src={book.imageURL}
-                alt={book.title}
-                width={book.dimensions.width}
-                height={book.dimensions.height}
-              />
-            </StyledImage>
+            <StyledImage
+              src={book.imageURL}
+              alt={book.title}
+              width={book.dimensions.width}
+              height={book.dimensions.height}
+            ></StyledImage>
           </div>
```

Comment:
Here you could reuse the `Book` (or in future the `BookImage` component), because if you look closely at the code you write it twice here.

Reply:
Hi @JessicaLoers thanks for your suggestions 🙂 Can you give me a hint how to do this? 🙏

Reply:
I just resolved it 😊 thnx